ansible/ansible.cfg

```diff
@@ -9,9 +9,24 @@
 # Set `ANSIBLE_CONFIG` for local changes
 
 [defaults]
+# Try more servers at once, smartly.
+forks = 20
+gathering = smart
+
+# Assume all actions take place under infrastructure/ansible
 roles_path = roles/
+inventory = ../../ansible/inventory/
+
+# These tend to be annoying.
 retry_files_enabled = False
 
-# Allow becoming an unprivileged user
-# http://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user
-allow_world_readable_tmpfiles = True
+# Deals with the files/ directory used by IDR
+inventory_ignore_extensions = ~, .orig, .bak, .ini, .retry, .pyc, .pyo, .html, .omero, .gif, .png, .js, .md
+
+# Helps determine what's running slowly
+callback_whitelist = profile_tasks
+
+# Performance options
+[ssh_connection]
+ssh_args = -o ForwardAgent=yes -o ControlMaster=auto -o ControlPersist=60s
```

Comment:
`-o ForwardAgent=yes` means your ssh agent will be forwarded to all servers you use with Ansible, is this necessary? (Without this `-o ControlMaster=auto -o ControlPersist=60s` should be the default)